po/com.ubuntu.calendar.pot

```diff
@@ -430,7 +430,7 @@ msgid "Show week numbers"
 msgstr ""
 
 #: ../qml/SettingsPage.qml:90
-msgid "Show lunar calendar"
+msgid "Display chinese calendar"
```

Comment:
Do you mind using upper case Chinese? Its more consistent in English then ;)
```suggestion
msgid "Display Chinese calendar"
```

Reply:
No I don't mind ;-P Thanks for the hint. Done.